services/graylog/docker-compose.yml

```diff
@@ -102,6 +102,9 @@ services:
       - /var/run/docker.sock:/var/run/docker.sock
     environment:
       - ALLOW_TTY=true
+      - MULTILINE_MATCH=first
+      - MULTILINE_PATTERN='^(ERROR|WARNING|INFO|DEBUG|CRITICAL):'
```

Comment:
Are you sure that the quotes ``'`` around the pattern are needed? I kind of recall that we had to remove them in the demo to make it work! 
Moreover, here since you use a list for the ``environment`` value, each entry is considered a string, i.e. ``"MULTILINE_PATTERN='^(ERROR|WARNING|INFO|DEBUG|CRITICAL):'"`` and when docker-compose splits into key-value, it will assign the quote symbol to the value!

Reply:
Yes, i had some problems to escape the : in the end of the string. Added [:] to find a workaround,